modules/server/src/main/scala/observe/server/ODBSequencesLoader.scala

```diff
@@ -20,7 +19,7 @@ import SeqEvent._
 import ObserveFailure.ObserveException
 import ObserveFailure.UnrecognizedInstrument
 
-final class ODBSequencesLoader[F[_]: ApplicativeError[*[_], Throwable]](
+final class ODBSequencesLoader[F[_]: Async](
```

Comment:
Weird that you need this change, What does need `Async`?

Reply:
There are some calls inside the code that require `Temporal`, and others that require `Sync`. `Async` groups both of them.